### PR TITLE
Fix for commands regarding missing product updates cause of bulk size…

### PR DESCRIPTION
**Pull request by PatrickJuergens**
In both commands
- bin/console sylius:elastic-search:reset-product-index --force
- bin/console sylius:elastic-search:update-product-index

there is a commit bulk size of 100 documents. If there are less than 100 products or even a rest than there is no commit for these product documents.

### CODE REVIEW

**Comment by JanTvrdik** [comment]
Isn't this already handled by `$this->elasticsearchManager->commit();` after the foreach?

**Comment by PatrickJuergens** [comment]
Yes, you are right there is a commit after the foreach at ResetProductIndexCommand. Sorry my fault, i will fix it. But there is no commit at the end of the UpdateProductIndexCommand

File: src/Command/ResetProductIndexCommand.php [hunk 1]
```diff
@@ -111,4 +111,4 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $output->writeln(sprintf('<info>Command is already running</info>'));
         }
     }
-}
+}
```
Comment (pamil) [inline src/Command/ResetProductIndexCommand.php]: Can you revert this change?

**Review by pamil — DISMISSED** [review]

File: src/Command/ResetProductIndexCommand.php (hunk 1 above)
Comment (PatrickJuergens) [reply src/Command/ResetProductIndexCommand.php]: Yes

Comment (11mb) [reply src/Command/ResetProductIndexCommand.php]: Isn't this quite complicated: introducing an extra variable to check wether there are still productDocument waiting? In the ResetProductIndexCommand this problem is solved by adding `$this->elasticsearchManager->commit();` after the foreach. I guess it could be possible that in some cases commit will be called twice.

**Review by pamil — APPROVED** [review]

**Comment by pamil** [comment]
Looks like Travis build finally got restarted :)

**Comment by pamil** [comment]
Thanks @PatrickJuergens! 🎉
